packages/shared/lib/services/notification/webhook.service.ts

```diff
@@ -106,9 +109,9 @@ class WebhookService {
         activityLogId: number,
         environment_id: number
     ) {
-        const { send, webhookInfo } = await this.shouldSendWebhook(nangoConnection.environment_id);
+        const { webhookInfo } = await this.shouldSendWebhook(nangoConnection.environment_id);
 
-        if (!send || !webhookInfo) {
+        if (!webhookInfo) {
```

Comment:
Why not checking `send` instead of `webhookInfo`
I find it error prone to have result type that are not unions. One cannot trust that the boolean attributes is enough to carry on with the logic.